ci/gather_diagnostics_data.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status, or zero if all
+# commands in the pipeline exit successfully.
+set -eu
+
+readonly SCRIPT_DIRECTORY="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+export TTY_OPTS="-t"
+
+if [ "${#}" -eq 0 ]; then
+  echo "The following arguments are required: services."
+  exit 1
+fi
+
+echo "Run data services diagnostic: $@"
+
+"${SCRIPT_DIRECTORY}"/ci_cli.py --type=diagnostic "$@"
```

Comment:
This file is so short now that I think we call `ci_cli.py` directly in TeamCity now and get rid of it altogether.

Reply:
README.md added